template-parts/header/builder/components.php

```diff
@@ -113,6 +113,14 @@
 		<?php
 		break;
 
+	case 'mode-switcher':
+		?>
+			<div class="ast-builder-layout-element site-header-focus-item ast-header-mode-switcher" data-section="section-mode-switcher">
```

Comment:
```suggestion
			<div class="ast-builder-layout-element site-header-focus-item ast-header-mode-switcher" data-section="section-mode-switcher">
```
@imnavanath Can this CSS class be removed? - `site-header-focus-item`

Reply:
Yes added this class conditionally, because it needed in customizer for shortcut edit-pencil icon.